src/BusTicket.java

```diff
@@ -1,13 +1,92 @@
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.List;
+
 class BusTicket {
-    String ticketClass;
-    String type;
-    String startDate;
-    double price;
+    private final String ticketClass;
```

Comment:
why these fields final?

Reply:
fields are marked as final to make sure the ticket details can’t be changed after the object is created, and for this project version it's meaningful for me, but maybe in cases when i would like to change tickets i should not use final